webapp/src/pages/DepositPage/EarnPage.tsx

```diff
@@ -327,11 +330,10 @@ const EarnPage = () => {
   const { vaultAccounts } = useVaultAccounts(vaultVersion);
   const vaultAccount = vaultAccounts[vaultOption || VaultList[0]];
   const Logo = getAssetLogo(asset);
-
   const isLoading = status === "loading" || loading;
   useRedirectOnSwitchChain(getChainByVaultOption(vaultOption as VaultOptions));
 
-  let logo = <Logo height="100%" />;
+  const logo = <Logo height="100%" />;
```

Comment:
why is this stored in a variable instead of used directly?

Reply:
it used is twiced so it was refactored

Reply:
gotcha